.github/workflows/test.yml

```
@@ -33,7 +33,7 @@ jobs:
 
       - name: Run tests with Pytest
         run: |
-          coverage run -m pytest
+          coverage run -m pytest -p no:warnings
```

```suggestion
          coverage run -m pytest
```

Use `with pytest.warns` to check if the warning is raised properly instead.